package.json

```diff
@@ -56,7 +56,7 @@
   ],
   "peerDependencies": {
     "@mermaid-js/mermaid-cli": ">= 8.13.7 < 10",
-    "braid-design-system": ">= 31.0.0",
+    "braid-design-system": ">= 31.14.0",
```

Comment:
Ah, should've technically marked this as breaking, but it wouldn't be a big deal, just a minor mismatch in the palettes